### PR TITLE
Profiling updates

### PR DESCRIPTION
**Pull request by aelanman**
## Description

Updates to the profiling wrapper scripts for new infrastructure. Some of these wrappers are very specific to my setup on Oscar and should probably be removed from the repository, but I'll leave that to discussion. Also a small change that allows the counter to be run on non-zero ranks, and it will attempt to estimate remaining runtime in the task loop.

## Motivation and Context

These wrappers are used for profiling, and haven't been updated in a year.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Reference simulation update or replacement
- [ ] Documentation change (documentation changes only)
- [ ] Version change
- [ ] Build or continuous integration change

## Checklist:

Bug fix checklist:
- [x] My fix includes a new test that breaks as a result of the bug (if possible).
- [x] All new and existing tests pass in both python 2 and python 3.
- [x] I have checked that I reproduce the reference simulations or if there are differences they are explained below (if appropriate). If there are changes that are correct, I will update the reference simulation files after this PR is merged.
- [x] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvsim/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by aelanman** [comment]
@bhazelton This is failing on travis in python3 only. The failure is in a test that looks like it's part of the spherical rotation methods added in the last PR.

**Comment by bhazelton** [comment]
hmm, this is actually something @jaguirre wrote, maybe he'll have some insight.

Did you try re-running it?

**Comment by aelanman** [comment]
I did :/

**Comment by aelanman** [comment]
@bhazelton This should be ready now.

File: environment.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: pyuvsim
 channels:
+  - astropy
   - conda-forge
   - defaults
```
[the remark this answers is not in the document]
Comment (aelanman) [reply environment.yml]: Okay, I've reset that.

**Pull request merged**